### PR TITLE
Cleaned up Dockerfile

### PR DESCRIPTION
**Pull request by leahnp**
added .dockerignore, added buildDeps var to Dockerfile and ordered deps alphabetically, changed base distro

**What this PR does / why we need it**:
Reduced docker image size from 124.2 MB to 105.1 MB.
Reduced total image security vulnerabilities from 81 to  57 
Reduced "high" level vulnerabilities from 10 to 0
Reduced "medium" level vulnerabilities from 26 to 21

https://github.com/samsung-cnct/issues/issues/71 that is currently blocked until the new CI is done will continue this work. Line 62 of the Dockerfile in this PR tries to manually cleanup artifacts but the multi-stage builds are a better fix for the long-term. 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: 
Related to: https://github.com/samsung-cnct/issues/issues/68

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.DS_Store
+.git
+.github
+.gitignore
+CODEOWNERS
+Dockerfile
```
Comment (davidewatson) [inline .dockerignore]: I am surprised you can exclude the Dockerfile itself from the context. I will open an issue against solas to create a default [`.dockerignore`](https://docs.docker.com/engine/reference/builder/#dockerignore-file).

File: Dockerfile [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
-		libfontconfig \
-		libfreetype6 \
-	--no-install-recommends && rm -rf /var/lib/apt/lists/*
+RUN buildDeps='apt-transport-https 
+			libfontconfig 
+			libfreetype6
+			wget' && \
+    apt-get -qq update && \
+    apt-get install -y -qq $buildDeps ca-certificates --no-install-recommends && \
+    rm -rf /var/lib/apt/lists/*
```
Comment (davidewatson) [inline Dockerfile]: CI is failing because this string is not terminated.

Comment (leahnp) [reply Dockerfile]: @davidewatson what string is that?

Comment (leahnp) [reply Dockerfile]: Sorry, I just realized I should look at the CI myself

Comment (davidewatson) [reply Dockerfile]: The logs say:

```
[91m/bin/sh: 1: Syntax error: Unterminated quoted string
[0mThe command '/bin/sh -c buildDeps='apt-transport-https' returned a non-zero code: 2
```

Unless you are trying to remain consistent with another Dockerfile, I would follow kraken-tools by declaring an environment variable in all caps and then referencing it: https://github.com/samsung-cnct/kraken-tools/blob/master/Dockerfile#L49

**Pull request merged**